mne/cuda.py

```diff
@@ -80,6 +81,30 @@ def init_cuda(ignore_config=False, verbose=None):
     logger.info('Enabling CUDA with %s available memory' % get_cuda_memory())
 
 
+@verbose
+def set_cuda_device(device_id, verbose=None):
+    """Set the CUDA device temporarily for the current session.
+
+    Parameters
+    ----------
+    device_id : int
+        Numeric ID of the CUDA-capable device you want MNE-Python to use.
+    %(verbose)s
+    """
+    if not _cuda_capable:
+        warn('CUDA is not enabled; did you forget to run `init_cuda()`?')
```

Comment:
either run ... or set MNE_USE_CUDA=true in your config?

And then this function should probably do the auto-init if `MNE_USE_CUDA` is true. Basically that config var should mean "I want you to automatically initialize when necessary".